.github/workflows/build-image.yml

```diff
@@ -72,3 +72,65 @@ jobs:
             org.opencontainers.image.revision=${{ github.sha }}
       - name: Echo image digest
         run: echo ${{ steps.docker_build.outputs.digest }}
+  rocm-image:
+    name: Build ROCm image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Free disk space
+        run: |
+          mkdir /tmp/emptydir
+          for dir in /usr/share/dotnet /usr/local/lib/android /opt/ghc; do
+            sudo rsync -a --delete /tmp/emptydir/ ${dir}
+          done
+          sudo apt-get clean
+          df -h
+          echo 'nproc: '$(nproc)
+      - name: Prepare metadata
+        id: metadata
+        run: |
+          DOCKER_IMAGE=superbench/superbench
+          IMAGE_TAG=dev-rocm4.2
```

Comment:
Use `dev-rocm4.2-pytorch1.7.0`? AMD will release torch1.7 and torch1.8 images.

Reply:
will we support both at the same time?

Reply:
Yes, we may run both in the future.

Reply:
what if there's pytorch 1.7, 1.8, 1.9, 1.10, ... for each rocm version?
I don't think we can support so many rocm images and create so many criteria for every version combination.

Reply:
Not exactly. We don't need to support so many ROCm images, for ROCm4.2 and 4.3, we just need to support 1.7 and 1.8.
For this case, how to you distinguish the image?

Reply:
> we just need to support 1.7 and 1.8
why do you choose 1.7 and 1.8 but do not choose 1.9?

Reply:
I think we have discussed in previous scrum? Copied from our slides:
```
SuperBench v0.2 AMD docker consolidation
ROCm 4.0 / 4.2
PyTorch 1.7.0 / 1.8.0
```

Reply:
No matter what pytorch version to provide, we should have the mechanism to distinguish images with same driver and different pytorch which will affect the performance a lot. They are different problem:
1. what pytorch version to provide, it's decided by us
2. mechanism to distinguish images.

We should firstly support 2, then to handle 1 according to the requirement.

For your change, we can only provide one image for specific ROCm version.